srv/adapter/openai.ts

```diff
@@ -20,11 +20,13 @@ export const handleOAI: ModelAdapter = async function* ({
   sender,
   log,
 }) {
+  log.info(`ASDASD`)
```

Comment:
Just remove these logs and we're good to go.

Reply:
my bad lol